ui/app/components/replication-page/replication-toggle.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  // ARG TODO: if other components need access to this response, will need to move it higher up to replication parent.
+  checkedValue: false,
+  handleToggleOff: computed('checkedValue', function() {
```

Comment:
since `handleToggleOff` is an event handler i don't think it should be a computed property. i think it makes sense to separate the concerns of 1) calling an event handler to pass updates up the chain and 2) presenting a value to the user. in this case, you could have `handleToggleOff` handle #1 while `checkedValue` handles #2. does that make sense?

Reply:
Totally!  Yeah, just getting it setup for a computed property, but it doesn't need to be at all.  It was almost more testing to make sure I knew who had access and control over what.  I'll remove.